renovate.json

```
@@ -29,6 +30,12 @@
 			"matchUpdateTypes": ["minor", "patch", "pin", "digest"],
 			"groupName": "other updates",
 			"enabled": true
+		},
+		{
+			"matchPackageNames": ["@types/three", "org.sonarsource.java:sonar-java-plugin", "three"],
+			"matchUpdateTypes": ["minor", "patch", "pin", "digest"],
```

Shouldn't major be also in this list?
```suggestion
			"matchUpdateTypes": ["major", "minor", "patch", "pin", "digest"],
```

---

I did not check but I thought major would be handled separately anyway or where is that configured for our other dependencies?

---

Because there is no explicit configuration for major PRs Renovate opens a separete one for every major update.
So it would make sense to me to cover those updates in this new (disabled) group also.

---

By the way, `matchUpdateTypes` (with major included) would match all update types. Thus, I assume we can omit it.

---

Renovate will consider by default all types
```suggestion
```